src/commands/dev/dev.ts

```diff
@@ -24,7 +24,7 @@ export default async function dev(
   );
 
   const [dir = '.'] = args;
-  const cwd = path.join(process.cwd(), dir);
+  const cwd = path.isAbsolute(dir) ? dir : path.join(process.cwd(), dir);
```

Comment:
This can just be:

```suggestion
  const cwd = path.resolve(dir);
```

`.resolve()` resolves against `process.cwd()` internally.